internal/go-sso/oidcauth/oidc.go

```diff
@@ -56,6 +56,10 @@ func (a *Authenticator) GetAuthCodeURL(ctx context.Context, redirectURI string,
 	authCodeOpts := []oauth2.AuthCodeOption{
 		oidc.Nonce(nonce),
 	}
+	// TODO: add resourceServer param here when it's required
```

Comment:
Is this TODO for this PR or the ambiguous future?

Reply:
it was a reminder as part of the original ticket from the client. I can remove it since we're not going to add this.